packages/regenesis-surgery/test/uniswap.spec.ts

```diff
@@ -50,6 +74,11 @@ describe('uniswap contracts', () => {
     })
 
     it('should have the same code as on mainnet', async () => {
+      if (!env.hasLiveProviders()) {
```

Comment:
I think this test should be runnable w/o a live provider as long as you have a connection to ETH?